packages/client/src/relewise.client.ts

```diff
@@ -72,14 +72,14 @@ export abstract class RelewiseClient {
             } catch (_) { 
             }
 
-            throw new ProblemDetailsError('Error when calling the Relewise API. Read more in the details property if there is error response or look in the network tab.', responseMessage);
+            throw new ProblemDetailsError(`Error when calling the Relewise API.\n\nTitle: ${response.statusText}\nStatus: ${response.status}\n\nRead more in the details property if there is error response or look in the network tab.`, responseMessage);
```

Comment:
If the `responseMessage` is set can we include the message `details.detail` so show even more information here?

Reply:
Sure thing good idea